wifi-subsys/components/network/src/httpsclient.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2022 Mesh4all <mesh4all.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @ingroup     wifi-subsys
+ * @{
+ * @file
+ * @brief      Configure a https client to make a request
+ *
+ * @copyright   Copyright (c) 2022 Mesh for all
+ * @author      RocioRojas <Rociorojas391@gmail.com>
+ *
+ */
+
+#include "esp_err.h"
+#include "esp_http_client.h"
+
+#ifndef HTTPSCLIENT_H
+#define HTTPSCLIENT_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Set task for https  get request
+ * Note: To create certificate in console:
+ * openssl s_client -showcerts -connect [URL]:443 </dev/null
+ *
+ * @return esp_err_t
+ */
+
+esp_err_t get_rest_funtion(int method);
+
+#ifdef __cplusplus
+}
+#endif
+#endif /* HTTPSCLIENT_H */
+/** @} */
```

Comment:
```suggestion
/** @} */

```
Add new line at EOF in each file